plugins/topology/src/utils/resource-utils.ts

```diff
@@ -288,3 +288,11 @@ export const getUrlForResource = (
     getIngressURLForResource(resources, resource)
   );
 };
+
+export const getCheCluster = (resources: K8sResponseData) => {
+  if (!resources.checlusters?.data?.length) return undefined;
+
+  return resources.checlusters.data.find(
+    cc => cc.metadata?.namespace === 'openshift-devspaces',
+  );
+};
```

Comment:
```
[].find(...) === undefined
```

Find on empty data already returns `undefined`. This whole function can be simplified to:

```suggestion
export const getCheCluster = (resources: K8sResponseData) => 
  resources.checlusters?.data?.find(
    cc => cc.metadata?.namespace === 'openshift-devspaces',
  );
```